pkg/subctl/cmd/logs.go

```diff
@@ -0,0 +1,63 @@
+package cmd
```

Comment:
Create a folder `subctl/cmd/gather` and rename this file to `connectivity.go` Change pkg name to `gather`. We'll have one file for each of the modules in this folder/pkg.